x/stake/types.go

```diff
@@ -116,23 +119,35 @@ func (c Candidate) validator() Validator {
 
 // Validator is one of the top Candidates
 type Validator struct {
-	Address     sdk.Address `json:"address"`      // Address of validator
-	VotingPower sdk.Rat     `json:"voting_power"` // Voting power if considered a validator
+	Address     sdk.Address   `json:"address"`
+	PubKey      crypto.PubKey `json:"PubKey"`
```

Comment:
Why does "PubKey" not follow the case spec of the other fields?

Reply:
whoops this is a typo good catch